Library/Homebrew/cli/named_args.rb

```diff
@@ -70,26 +70,29 @@ def to_formulae
           method:             T.nilable(Symbol),
           uniq:               T::Boolean,
           warn:               T::Boolean,
-        ).returns(T::Array[T.any(Formula, Keg, Cask::Cask)])
+        ).returns(T::Array[T.any(Formula, Cask::Cask)])
```

Comment:
I updated the return type to be consistent with the method name. It's safe according to the tests, and also let me improve error handling and remove three `T.cast`s elsewhere.

Reply:
It can be kegs from these call sites: https://github.com/Homebrew/brew/blob/673f19086b0909b13664b2a3873f26025ad5236c/Library/Homebrew/cli/named_args.rb#L226-L269

Reply:
Ah, good callout. I reverted the change, I'll revisit and maybe distinct methods to avoid having to do so many `cast`s.